app/assets/javascripts/components/instructor_panel.js.jsx

```diff
@@ -56,8 +66,15 @@ var InstructorPanel = React.createClass({
         <Action data={this.getInstructorToggleButtonData()} />
         <Action data={this.getTakeQueueOfflineButtonData()} />
         <Action data={this.emptyQueueButtonData()} />
-        
-
+        <div className="ui bottom padded fluid large menu">
+          <div className="ui simple dropdown item">
+            Merge Into Queue
+            <i className="dropdown icon"></i>
+            <div className="menu">
+              {mergeOptions}
+            </div>
+          </div>
+        </div>
```

Comment:
I noticed that the empty state (when there's only one queue in the course) for this is a little weird. Consider hiding it in that case?

Reply:
Done